io/net/sriov_device_test.py

```diff
@@ -250,11 +255,19 @@ def device_add_remove(self, slot, port, mac, logical_id, operation):
 
         if not self.backup_veth_vnetwork and not self.vnic_sriov_adapter and self.sriov_adapter:
             if operation == 'add':
-                cmd = 'chhwres -r sriov -m %s --rsubtype logport \
-                      -o a -p %s -a \"adapter_id=%s,phys_port_id=%s, \
-                      logical_port_type=eth,mac_addr=%s,migratable=%s%s\" ' \
-                      % (self.server, self.lpar, adapter_id,
-                         port, mac, self.migratable, backup_device)
+                if mac is None:
```

Comment:
why this check ?

Reply:
@abdhaleegit 
if this check is not enabled, then user must provide "N" number of MAC address with space separated for "N" number of sriov logical ports in YAML file
suppose user wanted to create 10 Number of SRIOV ports or more than that, user should be providing those many MAC addresses which will be trouble some.

There are two ways to create SRIOV ports on HMC Cli -

1. chhwres -r sriov -m ltcden4 --rsubtype logport   -o a -p ltcden4-lp2-Farooq -a "adapter_id=5,phys_port_id=1,                     logical_port_type=eth,mac_addr=16163e7793c1,migratable=0"  ----> with MAC address

2. chhwres -r sriov -m ltcden4 --rsubtype logport  -o a -p ltcden4-lp2-Farooq -a "adapter_id=5,phys_port_id=0 ---> without MAC address. 

Since our test case ( create max_logical_ports) required just a port, have enabled the second method to add logical_ports without worrying about mac address.

Reply:
@FarooqAbdulla02 perfect,  just add the same in documentation